.github/renovate.json

```diff
@@ -10,5 +10,6 @@
     "automergeType": "branch",
     "schedule": ["before 5am"]
   },
-  "rangeStrategy": "update-lockfile"
+  "rangeStrategy": "update-lockfile",
+  "commitMessagePrefix": ":package:"
```

Comment:
Is `:package:` replaced with the package name?

Reply:
Nop, it just add a :package: before each commit message

Reply:
🤣sorry!  I shouldn't comment after midnight cuz that's when I unplug my brain